include/ignition/utils/ImplPtr.hh

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright (C) 2018 Open Source Robotics Foundation
```

Comment:
```suggestion
 * Copyright (C) 2020 Open Source Robotics Foundation
```

?

Reply:
These were copied from @mxgrey 's implementation, which was originally written in 2018 :)

I also can't take credit for the style update for that reason.

Reply:
Ha gotcha, sounds good